### PR TITLE
ref(tunnel): Remove Axios in favor of built-in Fetch in Next.js

### LINKED ISSUE #103: Remove axios from tunneling/nextjs example
Nextjs has a `fetch` polyfill built in: https://nextjs.org/blog/next-9-4#improved-built-in-fetch-support.

_See original discussion in https://github.com/getsentry/examples/pull/102#r680967830_

### PR DESCRIPTION
**Pull request by kamilogorek**
Closes #103

### CODE REVIEW

File: tunneling/nextjs/pages/api/tunnel.js [hunk 1]
```diff
@@ -27,7 +26,13 @@ async function handler(req, res) {
     }
     
     const url = `https://${sentryHost}/api/${projectId}/envelope/`;
-    return await axios.post(url, envelope);
+    // Starting Next.js v9.4, Fetch API is polyfilled in both, Browser and Node.js environments
+    // https://nextjs.org/blog/next-9-4#improved-built-in-fetch-support
```
Comment (rhcarvalho) [inline tunneling/nextjs/pages/api/tunnel.js:30]:
```suggestion
```

File: tunneling/nextjs/pages/api/tunnel.js [hunk 2]
```diff
@@ -27,7 +26,13 @@ async function handler(req, res) {
     }
     
     const url = `https://${sentryHost}/api/${projectId}/envelope/`;
-    return await axios.post(url, envelope);
+    // Starting Next.js v9.4, Fetch API is polyfilled in both, Browser and Node.js environments
```
Comment (rhcarvalho) [inline tunneling/nextjs/pages/api/tunnel.js:29]:
```suggestion
```

**Review by rhcarvalho — COMMENTED** [review]
- Need to remove axios from package.json
- I'd move the commentary to the commit message. The example requires a much newer version of nextjs anyway, so we can leave the example code cleaner.

**Comment by kamilogorek** [comment]
Updated all dependencies just so we are using latest

**Pull request merged**